style/js/README.md

```diff
@@ -1,3 +1,163 @@
+JavaScript Style Guides
+=====
+
+Guides for programing JavaScript in style
+
+General
+---
+* Use [ESLint](http://eslint.org/)
+* Always use `use strict`
+
+Parens and Braces
+---
+* Leave one space between `if` and `(`.
```

Comment:
We should be consistent about appending a *period* mark at the end of a bullet, or not.